end_to_end/gpu/a3/test_oss_gpu_tests.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+set -uex
+
+# This script reflects the list of GPU tests in .github/workflows/UnitTests.yml
```

Comment:
Should we be merging this? I don't understand. Don't we want to model our tests as one test per step? (That makes them more isolated and reproducible.)

Reply:
Yes, I agree. I first thought that it'd be too tedious and verbose to add these tests to XLML as separate tests since I would have to write a script for each test. But I think I found a more concise way to do it. Please take a look the new change and this PR https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/283